lib/ansible/plugins/connection/ssh.py

```diff
@@ -398,6 +399,10 @@
 SSHPASS_AVAILABLE = None
 SSH_DEBUG = re.compile(r'^debug\d+: .*')
 
+tty_parser = argparse.ArgumentParser()
```

Comment:
he, i do love the use of argparse to deal with 'figuring out' the ssh args passed 
+1